.github/workflows/docker.yaml

```diff
@@ -67,4 +67,4 @@ jobs:
         if: ${{ github.event_name == 'workflow_dispatch' && github.ref_name != 'main' }}
         run: |
           docker tag $IMAGE_NAME:${{ github.sha }} $DOCKER_DEV/$IMAGE_NAME:${{github.event.inputs.tag}}
-          docker push $DOCKER_DEV/$IMAGE_NAME:${{github.event.inputs.tag}}
\ No newline at end of file
+          docker push $DOCKER_DEV/$IMAGE_NAME:${{github.event.inputs.tag}}
```

Comment:
e.g. @illusional

Reply:
This is adding the EOF line?

Reply:
Not sure I understand - the [plugin](https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#end-of-file-fixer) is described as 'Makes sure files end in a newline and only a newline.', and there's no newline here. I'm sure I'm wrong here, but not sure why.

Reply:
GitHub shows the inverse, it puts the red <anchor> where there isn't a newline (see the left diff) and hides the newline where there is (right): 
<img width="617" alt="image" src="https://github.com/populationgenomics/production-pipelines/assets/22381693/19170a13-3169-4b6f-8fd0-8cf9d124a518">

Reply:
how frustrating :D